api/lib/infrastructure/repositories/user-repository.js

```diff
@@ -286,6 +286,20 @@ module.exports = {
       });
   },
 
+  async isEmailAllowedToUseForCurrentUser(userId,email) {
+
+    const userFound = await BookshelfUser
+      .where('id', '!=', userId)
+      .where({ email: email.toLowerCase() })
+      .fetch();
+    if (userFound) {
+      throw new AlreadyRegisteredEmailError();
+    }
+    else {
+      return email;
+    }
```

Comment:
```suggestion
    return true;
```
Vu que la méthode de repository est de type `is...` je m'attends à ce que ça retourne un boolean